stdlib/2and3/genericpath.pyi

```diff
@@ -7,16 +7,21 @@ else:
     def commonprefix(m: Sequence[AnyStr]) -> AnyStr: ...
 if sys.version_info >= (3, 6):
     from os import PathLike
-    def exists(path: Union[AnyStr, PathLike[AnyStr]]) -> bool: ...
+    def exists(path: Union[AnyStr, PathLike[AnyStr], int]) -> bool: ...
+    def isfile(path: Union[AnyStr, PathLike[AnyStr], int]) -> bool: ...
+    def isdir(s: Union[AnyStr, PathLike[AnyStr], int]) -> bool: ...
+    def getsize(filename: Union[AnyStr, PathLike[AnyStr], int]) -> int: ...
+    def getmtime(filename: Union[AnyStr, PathLike[AnyStr], int]) -> float: ...
+    def getatime(filename: Union[AnyStr, PathLike[AnyStr], int]) -> float: ...
+    def getctime(filename: Union[AnyStr, PathLike[AnyStr], int]) -> float: ...
 else:
     def exists(path: Text) -> bool: ...
-def isfile(path: Text) -> bool: ...
-def isdir(s: Text) -> bool: ...
-def getsize(filename: Text) -> int: ...
-def getmtime(filename: Text) -> float: ...
-def getatime(filename: Text) -> float: ...
-def getctime(filename: Text) -> float: ...
-
+    def isfile(path: Text) -> bool: ...
```

Comment:
These need to take `Union[bytes, Text]` so they can accept both bytes and text in Python 3 < 3.6, consistent with how they were previously defined in `os.path`.